data/src/main/resources/kcsb.json

```diff
@@ -0,0 +1,218 @@
+{
```

Comment:
Quick suggestion so we can manage this easily.
Can we have a `"Version": "YYYY-MM-DD"` field that's only used to compare and align this file across repos?

Reply:
Great suggestion, I'll also update in python.